README_TR.md

```diff
@@ -0,0 +1,28 @@
+![](https://raw.github.com/HaxeFlixel/haxeflixel.com/master/src/files/images/flixel-logos/flixel-addons.png)
+
+[flixel](https://github.com/HaxeFlixel/flixel) | [uzantılar](https://github.com/HaxeFlixel/flixel-addons) | [ui](https://github.com/HaxeFlixel/flixel-ui) | [demolar](https://github.com/HaxeFlixel/flixel-demos) | [araçlar](https://github.com/HaxeFlixel/flixel-tools) | [şablonlar](https://github.com/HaxeFlixel/flixel-templates) | [dökümanlar](https://github.com/HaxeFlixel/flixel-docs) | [haxeflixel.com](https://github.com/HaxeFlixel/haxeflixel.com) | [orijinal](/README.md)
+
+[![CI](https://img.shields.io/github/actions/workflow/status/HaxeFlixel/flixel-addons/main.yml?branch=dev&logo=github)](https://github.com/HaxeFlixel/flixel/actions?query=workflow%3ACI)
+[![Discord](https://img.shields.io/discord/162395145352904705.svg?logo=discord)](https://discordapp.com/invite/rqEBAgF)
+[![Haxelib Version](https://badgen.net/haxelib/v/flixel-addons)](https://lib.haxe.org/p/flixel-addons)
+[![Haxelib Downloads](https://badgen.net/haxelib/d/flixel-addons?color=blue)](https://lib.haxe.org/p/flixel-addons)
+[![Haxelib License](https://badgen.net/haxelib/license/flixel-addons)](LICENSE.md)
+[![Patreon](https://img.shields.io/badge/donate-patreon-blue.svg)](https://www.patreon.com/haxeflixel) 
+
+
+
+## Hakkında
+
+[HaxeFlixel](https://github.com/HaxeFlixel/flixel) | [HaxeFlixel](https://github.com/HaxeFlixel/flixel) için kullanışlı ancak isteğe bağlı, topluluk tarafından oluşturulan bir takım sınıf seti.  [Flixel Power Tools](https://github.com/photonstorm/Flixel-Power-Tools)'tan bazı sınıflar dahildir.
```

Comment:
I assume it is on purpose that there is no space between "Power Tools" and "'tan"

Reply:
Yes, because its seperated with the " ' " sign